addons/binding/org.openhab.binding.systeminfo/ESH-INF/thing/computer.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="systeminfo" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- This thing represents a computer with single logical storage device,single display, battery and one network interface 
+		installed. If other configurations are needed feel free to create different thing types. Currently dynamic channel creation 
+		is not supported from the binding. -->
+	<thing-type id="computer">
+		<label>Systeminfo</label>
+		<description>Sample thing for systeminfo Binding</description>
+
+		<channels>
+			<channel id="os_manufacturer" typeId="os_manufacturer"></channel>
```

Comment:
how about using channel groups? This binding seems to be predestined for that :-)